content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -0,0 +1,327 @@
+
+### Integration Tests
+
+Integration tests are the next level up after unit tests. These higher-order tests focus on the interactions between multiple components and dependencies. Integration tests cover a slightly wider workflow, though they are exponentially broader in scope and far more sensitive to changes across the codebase. This means that they can catch many bugs, which is both a benefit and a drawback. The main drawbacks come from how slow these tests can be to run and debug. When a test chains multiple calls and verifications interacting with different concerns, any test failures will provide a more complex debugging experience.
+
+Integration tests are useful for areas of critical functionality, and for detecting failures which unit tests cannot catch. As you approach major functional milestones, add or update integration tests. As you identify features which emit many bugs, consider adding integration tests to more quickly detect issues. If you find an integration test frustrating to investigate and debug, consider changes to the underlying feature and its unit tests.
+
+Here are real-world examples of O3DE's integ tests in [C++](https://github.com/o3de/o3de/blob/development/Gems/EMotionFX/Code/Tests/MotionEventTrackTests.cpp) and [Python](https://github.com/o3de/o3de/blob/development/AutomatedTesting/Gem/PythonTests/largeworlds/dyn_veg/EditorScripts/RotationModifierOverrides_InstancesRotateWithinRange.py), which each require examining many sibling files. These tests take on the order of one minute to automatically verify.
+
+#### Mitigating Drawbacks
+
+1. To reduce overall debugging time, unit tests can act as a filter to catch simple issues before they appear as confusing integration test failures
+    * Investigate unit test failures first
+    * When an integration test failure is identified, take a moment to ask "how could this be caught with a unit test?"
+2. Tests should remain focused on the behavior of one specific feature
+    * Structurally, an integration test should try to follow the simple pattern of `Arrange, Act, Assert` and avoid taking many different actions
+    * While integration tests are intended to cover multiple dependencies, reducing the scope of actions and assertions reduces the risk of spending more time debugging tests than fixing the underlying problem
+3. Reduce the amount of code necessary for the integration test
+    * Consider refactoring the production interface to make it easier to write (unit) tests
+    * Remove similar highly-similar integration tests by combining their steps, and/or eliminating less-critical verifications
```

Comment:
it might be worth mentioning that unlike unit tests, Editor Python Binding in editor automation involves the launch and initialization of an editor instance. Whenever possible tests should utilize batched testing with the multitest framework and individual tests may consider containing more complex steps rather than breaking across multiple tests. While this diverges from the atomic isolation of tests it reduces overall execution time.

Reply:
I actually removed this advice because of recent issues with batched and parallel execution ...but as you mention, I it's still good advice to heuristically aim for. Existence of bugs shouldn't mean an area needs to be avoided! I think it's worth adding back in.